src/components/Backdrop/Backdrop.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import styled, { css } from 'react-emotion';
+import PropTypes from 'prop-types';
+import Fade from '../Fade';
+import { duration } from '../../styles/transitions';
+
+const baseStyles = css`
+  z-index: -1;
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  -webkit-tap-highlight-color: transparent;
+  background-color: rgba(0, 0, 0, 0.5);
+`;
+
+const visibilityStyles = ({ invisible }) => css`
+  ${invisible && 'background-color: transparent'};
+`;
+
+const Shadow = styled.div`
+  ${baseStyles} ${visibilityStyles};
+`;
+
+function Backdrop(props) {
+  const { className, invisible, open, transitionDuration, ...other } = props;
+
+  return (
+    <Fade appear in={open} timeout={transitionDuration} {...other}>
+      <Shadow invisible={invisible} aria-hidden="true" />
```

Comment:
I'm not sure the `aria-hidden` is necessary here since the backdrop doesn't contain any text and doesn't seem to be focusable. What was your intention behind adding it?

Reply:
It was not my intention. All the components I've added are ported from `material-ui` component library to styled components and circuit-ui.

`aria-hidden-true` SHOULD NOT be used on a focusable visible element, which `shadow` is not:

https://www.w3.org/TR/using-aria/#fourth

Also, from the above link:
*If an interactive element cannot be seen or interacted with, then you can apply aria-hidden*

...which makes complete sense for that element.